### DISCUSSION TITLE
Real time events have stopped loading in the UI when 'tail' enabled.

### DISCUSSION THREAD

**Discussion by dom-advancedcommerce** [datalust/seq-tickets#2322] [Help]
We have just upgraded from 'v2021.2.6459' to 'v2024.3.12680'.

The real time events have stopped loading in the UI when the 'tail' icon is enabled. This is the most important element of SEQ for us. 

Now we have to F5, to see new events. What could be the cause of this (setting/bug), and how can we resolve.

Image attached showing 'tail' enabled
![seqtail](https://github.com/user-attachments/assets/8be018ad-9ddd-4e3b-8c2f-33180131216f)

**Comment by nblumhardt** [comment]
Hi Dom, thanks for your message.

With Tail enabled, are any events appearing in the browser's developer tools "Network" or "Console" tabs? If so, could you please send some details of what you find there.

Otherwise, please let us know what browser and operating system you're using, and whether you have any WAF/firewall/proxy between your browser and Seq.

Best regards,
Nick

**Comment by dom-advancedcommerce** [comment]
Hi, we identified the issue

In the new version the tail signal request url has indeed changed which hit our FW.

**Reply by nblumhardt** [reply]
That's great; thanks for letting us know, @dom-advancedcommerce 👍
